src/main/java/com/code_intelligence/jazzer/driver/Driver.java

```diff
@@ -86,6 +87,13 @@ public static int start(List<String> args, boolean spawnsSubprocesses) throws IO
     }
 
     // Do not modify properties beyond this point, loading Opt locks in their values.
+    if (Opt.instrumentOnly) {
```

Comment:
I went back and forth on this. I wanted to put this before so that the Offline instrumentation method could live on it's own and all you had to do was call that method with minimal setup before calling the offline instrumentation function (not that I really expect this method to be called from anywhere else). 

I'll move this to below the AgentInstall.install though, that makes a lot of sense.

Reply:
On second thought I think I will leave this inside of instrumentJars, otherwise this function may or may not do what you want it to do, depending on if AgentInstall.install was called or not.

I also want to create a build that doesn't use Driver.java, that is only for offline instrumentation and doesn't have the fuzzing component. I want to try to take this more lightweight instrumentation only tool and include it in some build systems so that we can have an instrumented Android jar/apk at compile time. That's the goal at least.